environments/templates/alert-stream-broker-application.yaml

```diff
@@ -18,6 +18,13 @@ spec:
   destination:
     namespace: alert-stream-broker
     server: https://kubernetes.default.svc
+  ignoreDifferences:
+  #- jsonPointers:
+    #- /spec/volumeName
+    #- /spec/storageClassName
```

Comment:
Is this being left in for future use?

Reply:
Thanks!  Yes.  Will do separate PR to fix out of sync argocd issue.